<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryMainWidget</class>
 <widget class="QWidget" name="HistoryMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>540</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="multiple_day">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Geneva</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 120, 120);</string>
     </property>
     <property name="text">
      <string>Multiple Day Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="single_day">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Geneva</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 120, 120);</string>
     </property>
     <property name="text">
      <string>Single Day Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>95</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/files/timer_clock_time_vector_symbol-512.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>572</x>
       <y>134</y>
       <width>95</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/files/calendar.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
